README.md

```diff
@@ -29,15 +29,20 @@ Why?
 
 ## Usage
 
-To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
+### Dependencies
 
-    $ sudo gem install jekyll bundler
+Install [asdf-vm](https://asdf-vm.github.io/asdf/#/core-manage-asdf-vm) for your OS and now using asdf-vm, install [ruby](https://github.com/asdf-vm/asdf-ruby).
+
+### Cloning and running
+
+To clone the repository and run this website on your local machine, type the following commands:
```

Comment:
Instead of this, infom the user to clone the repository followed by the clone command.
```
Clone the coala Projects repository
git clone <link to repository>
```
and then the steps to run the website
```
Run coala Projects website
$ cmd1
$ cmd2
```

Reply:
> Clone the coala Projects repository

I think `To clone the repository, type the following command` would look much better.